src/main/java/com/sap/oss/phosphor/fosstars/data/github/BanditDataProvider.java

```diff
@@ -0,0 +1,119 @@
+package com.sap.oss.phosphor.fosstars.data.github;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.RUNS_BANDIT_SCANS;
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.USES_BANDIT_SCAN_CHECKS;
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.setOf;
+
+import com.sap.oss.phosphor.fosstars.data.AbstractStaticScanToolsDataProvider;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.ValueSet;
+import com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.GitHubProject;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.oss.phosphor.fosstars.util.Yaml;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import org.apache.commons.collections4.IteratorUtils;
+
+/**
+ * The data provider gathers info about how a project uses Bandit for static analysis. In
+ * particular, it tries to fill out the following features:
+ * <ul>
+ *   <li>{@link OssFeatures#RUNS_BANDIT_SCANS}</li>
+ *   <li>{@link OssFeatures#USES_BANDIT_SCAN_CHECKS}</li>
+ * </ul>
+ */
+public class BanditDataProvider extends AbstractStaticScanToolsDataProvider {
+
+  /**
+   * A step in a GitHub action that triggers analysis with Bandit.
+   */
+  private static final Pattern RUN_STEP_BANDIT_REGEX_PATTERN
+      = Pattern.compile("^.*bandit .*$", Pattern.DOTALL);
```

Comment:
I guess, the whitespace is here for a reason.

Reply:
yes, the syntax goes `bandit -r`. Just to indicate that bandit is a function call during `run:` in GitHub actions and not randomly occuring command word. Maybe I should include all the parameters possible which comes with bandit run.

However to improve this, let me work on this here #789 789